packages/kanban/src/board/indexer.ts

```diff
@@ -1,11 +1,12 @@
 import { readFile, writeFile } from "node:fs/promises";
 import path from "node:path";
+import { fileURLToPath } from "node:url";
```

Comment:
For the CLI check below, compare canonical paths: `realpath(entry) === realpath(fileURLToPath(import.meta.url))`. `pnpm` links and `node --preserve-symlinks` can make the raw `resolve(...)` equality lie.